tests/test_upload.py

```diff
@@ -72,6 +73,27 @@ def test_successs_prints_release_urls(upload_settings, stub_repository, capsys):
     assert captured.out.count(NEW_RELEASE_URL) == 1
 
 
+def test_successs_prints_uploaded_package_names_if_verbose(upload_settings, capsys):
+    """Prints the path of each distribution attempting to be uploaded."""
```

Comment:
Looks like you attempted to follow the naming/docstring conventions in the tests below, which is generally a good idea. However, I think think is more accurate/descriptive, and follows are recent agreement to use imperative mood in docstrings:

```suggestion
def test_print_packages_if_verbose(upload_settings, capsys):
    """Print the path and file size of each distribution attempting to be uploaded."""
```

I fixed the mood in the existing tests in #657.